hack/update-csv.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+import logging
+import sys
+import yaml
+
+
+_ANNOTATIONS = {
+    'categories': 'Openshift Optional',
+    'description': \
+        'Manages KubeVirt addons for Scheduling, Scale, Performance',
+    'containerImage': 'REPLACE_IMAGE:TAG',
+}
+_VERSIONED_NAME = 'ssp-operator.vPLACEHOLDER_CSV_VERSION'
+_DESCRIPTION = "KubeVirt Schedule, Scale and Performance Operator"
+_NAMESPACE = 'kubevirt'
+_SPEC = {
+    'description': _DESCRIPTION,
+    'provider': {
+        'name': 'KubeVirt project'
+    },
+    'maintainers': [{
+        'name': 'KubeVirt project',
+        'email': 'kubevirt-dev@googlegroups.com',
+    }],
+    'keywords': [
+        'KubeVirt', 'Virtualization', 'Template', 'Performance',
+        'VirtualMachine', 'Node', 'Labels',
+    ],
+    'links': [{
+        'name': 'KubeVirt',
+        'url': 'https://kubevirt.io',
+    }, {
+        'name': 'Source Code',
+        'url': 'https://github.com/kubevirt/ssp-operator'
+    }],
+    'labels': {
+        'alm-owner-kubevirt': 'ssp-operator',
+        'operated-by': 'ssp-operator',
+    },
+    'selector': {
+        'matchLabels': {
+            'alm-owner-kubevirt': 'ssp-operator',
+            'operated-by': 'ssp-operator',
+        },
+    },
+    'version': 'vPLACEHOLDER_CSV_VERSION'
+}
```

Comment:
It looks like many of these changes could be applied directly to `config/manifests/base/ssp-operator.clusterserviceversion.yaml`.
Then this code can be simplified.